substrate/frame/balances/src/benchmarking.rs

```diff
@@ -31,7 +31,13 @@ const SEED: u32 = 0;
 // existential deposit multiplier
 const ED_MULTIPLIER: u32 = 10;
 
-const MINIMUM_BALANCE: u32 = 100;
+fn minimum_balance<T: Config<I>, I: 'static>() -> T::Balance {
+	if cfg!(feature = "insecure_zero_ed") {
+		100u32.into()
+	} else {
+		T::ExistentialDeposit::get()
+	}
```

Comment:
Don't we want the reverse? I am probably confusing things

Reply:
If we have `feature = "insecure_zero_ed"`:
- It would mean that we have zero as `T::ExistentialDeposit`
- So we change it to 100

Otherwise:
- We know that the ED is not zero
- So we use it as is